### DISCUSSION TITLE
New Chart Release?

### DISCUSSION THREAD

**Discussion by erie149** [apache/airflow#42584] [General]
Was wondering if a new chart release will happen soon?  I would like to move to airflow 2.10.x.  I know we could do it via the latest helm chart in main but I think it would be wise to have an official chart release for the 2.10.x airflow version.

**Comment by razoreater** [comment]
Any news about a new chart release?
Currently we are using chart version 1.15.0 but we would like to move to airflow 2.10.x.

**Comment by potiuk** [comment]
> Currently we are using chart version 1.15.0 but we would like to move to airflow 2.10.x.

You can move any time. The release of chart is independent from Airflow version and you are absolutely free to use Airflow 2.10 with the chart version that you use. The "Airflow version" setup in chart is merely a "default" and it's absolutely on your side to upgrade to a newer version of Airflow by modifying your values there.

**Comment by potiuk** [comment]
>  I know we could do it via the latest helm chart in main but I think it would be wise to have an official chart release for the 2.10.x airflow version.

There is absolutely no reason to wait for it.

**Comment by atzannes** [comment]
In case this is of interest to anyone, yesterday I performed an upgrade of my airflow deployment from version 2.9.3 -> 2.10.0 -> 2.20.1 -> 2.10.2 -> 2.10.3 -> 2.10.4 (which is the latest as of yesterday) using the current Helm chart (airflow-1.15.0), by updating my values.yaml file to include the following two lines

```
airflowVersion: "2.10.4"
defaultAirflowTag: "2.10.4"
```
I started only with airflowVersion but it was not sufficient.

I did it step by step to be extra carefull, but everything went seamlessly.

**Reply by razoreater** [reply]
Can you give an example of your values,yaml file @atzannes?
I'm looking to upgrade.

**Reply by atzannes** [reply]
The above two lines are literally all I changed. They are at the top two lines the `values.yaml`. Everything after that is literally setting up the DB, ingress, and things specific to my prod deployment which are non-transferable. For the most part I followed the Helm chart docs for those other parts.

**Reply by razoreater** [reply]
Managed to get It working, thanks @atzannes 🥳
